domains/authorization/Authorization.API/API/Authorization/Controllers/TermsController.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Threading.Tasks;
+using API.Authorization._Features_;
+using Asp.Versioning;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Authorization.Controllers;
+
+[ApiController]
+[ApiVersion(ApiVersions.Version20230101)]
+[Route("api/authorization/terms")]
+public class TermsController(IMediator mediator) : ControllerBase
+{
+    [ProducesResponseType(typeof(TermsResponseDto), StatusCodes.Status200OK)]
+    [HttpPost("accept")]
+    public async Task<IActionResult> AcceptTerms([FromBody] AcceptTermsDto acceptTermsDto)
```

Comment:
I would change this endpoint only to create organization and set terms version to whatever version we have. Not doing anything with user. Then let Get consent endpoint create / update user. So we don't have duplicate places that does same logic.